config/database.php

```diff
@@ -48,7 +48,7 @@
 
         'sqlite' => [
             'driver' => 'sqlite',
-            'database' => env('DB_DATABASE', database_path('database.sqlite')),
+            'database' =>  storage_path() . DIRECTORY_SEPARATOR . env('DB_DATABASE','database') . '.sqlite',
```

Comment:
why you did that

Reply:
loading the sqlite from the storage folder not database cause database is tracked by git